website/static/js/registrationUtils.js

```diff
@@ -263,7 +263,8 @@ var MetaSchema = function(params) {
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid] = new Question(question, qid);
+            var questionId = question.qid;
```

Comment:
Looks like this is a remnant from previous way of doing it. It probably should be able to be

```js
mapped[qid] = new Question(question, qid);
```